tests/manifests/test_configs_consistency.py

```diff
@@ -0,0 +1,709 @@
+                else:
+                    current_source_of_mount = MountedEmptyDir.from_template(
+                        current_volume["name"], volume_mount, deployable_details.content_volumes_mapping
+                    )
+                validated_config.mutable_empty_dirs[current_volume["name"]] = current_source_of_mount
+            elif "persistentVolumeClaim" in current_volume:
+                current_source_of_mount = MountedPersistentVolume.from_template(
+                    current_volume, volume_mount, deployable_details.content_volumes_mapping
+                )
+            # If we have a subPath we filter the files using a SubPathMount
+            if "subPath" in volume_mount:
+                validated_config.sources_of_mounted_paths.append(
+                    SubPathMount(volume_mount["subPath"], current_source_of_mount)
+                )
+            else:
+                validated_config.sources_of_mounted_paths.append(current_source_of_mount)
+
+        if is_matrix_tools_command(container_spec, "render-config"):
+            render_config_consumer = RenderConfigContainerPathConsumer.from_container_spec(
+                container_spec,
+                workload_spec,
+                templates,
+                other_secrets,
+                validated_config.mutable_empty_dirs,
+            )
+            validated_config.paths_consumers.append(render_config_consumer)
+        else:
+            validated_config.paths_consumers.append(
+                GenericContainerSpecPathConsumer.from_container_spec(
+                    workload_spec, container_spec, previously_mounted_empty_dirs
+                )
+            )
+        return validated_config
+
+    def check_mounted_files_unique(
+        self, skip_path_consistency_for_files, ignore_unreferenced_mounts, ignore_paths_mismatches
+    ):
+        mounted_files = [
+            node_path(parent_mount, mount_node)
+            for source in self.sources_of_mounted_paths
+            for parent_mount, mount_node in source.get_mounted_paths()
+        ]
+        assert len(mounted_files) == len(set(mounted_files)), (
+            f"{self.template_id}/{self.name} : "
+            f"Mounted files are not unique \n"
+            f"Duplicated files : { {item for item, count in Counter(mounted_files).items() if count > 1} }\n"
+            f"From Mounted Sources : {self.sources_of_mounted_paths}"
+        )
+
+    def check_paths_used_in_content(
+        self, skip_path_consistency_for_files, ignore_unreferenced_mounts, ignore_paths_mismatches
+    ):
+        paths_not_found = []
+        skipped_paths = []
+        for source in self.sources_of_mounted_paths:
+            for parent_mount, mount_node in source.get_mounted_paths():
+                if (
+                    node_path(parent_mount, mount_node) in ignore_unreferenced_mounts.get(self.name, [])
+                    or parent_mount.path.startswith("/secrets")
```

Comment:
Can this be documented like the `/conf` ignore is elsewhere?

Is it possible to push trhis down into a specific `PathConsumer` or does it apply to all?